src/rdf4cpp/rdf/datatypes/registry/FixedIdMappings.hpp

```diff
@@ -13,6 +13,7 @@ namespace rdf4cpp::rdf::datatypes::registry {
 using storage::node::identifier::LiteralType;
 
 inline constexpr util::ConstexprString default_graph_iri{""};
+inline constexpr util::ConstexprString rdf_type{"http://www.w3.org/1999/02/22-rdf-syntax-ns#type"};
```

Comment:
Regarding this issue: I talked with @bigerl and we agreed that it probably does not make sense to fix the underlying problem in this PR because it would be a slightly larger fix and the problem has not much to do with this PR.

Could you please add a TODO (and create an issue) that reminds us that we need to refactor this at some point?

Reply:
Quickly created them myself in the interest of getting this merged